### PR TITLE
Partners

### LINKED ISSUE #351: Partners page and home copy update
## Changes request
- Create new Partners page with design, photos, content on Figma.
- Change last graf in home page blurb and add images:
image

- Use Contact template from #355 

## New styles
- Pull quote link style
- One new text/link style to be added.

## Special instructions
- Deploy this branch separately so that it can be sent to Partners for approval

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #351 

## What this PR does
- [x] Home page: Add image logos
- [x] Partners page: Contact template
- [x] Partners page: Add photos, text, links
- [x] CSS: Add pulled quote text style
- [x] CSS: Add statistics text style
- [x] CSS: Add small attribution text style
- [x] CSS: All the spacings - Desktop
- [x] CSS: All the spacings - Mobile
- [x] Design: Add statistics box rhombus style

## Notes
- This PR changed the names for _all_ the CSS classes/SVG paths with rhombuses - that includes the homepage, Press, Resources and Partners page. So please review all the pages for any regressions (there shouldn't be though!).
- I'm starting this new thing of having a basic naming convention for images: `(name of page)-(type of image)-(slug)`, like `home-logo-calsta` or `partners-background-cdt`. Mobile only images have `--sm` appended at the end:
image

### CODE REVIEW

**Comment by machikoyasuda** [comment]
This PR is ready for a code review. The copy is not finalized yet, so I have just used the latest available copy.

I also filed #365 this because I noticed the blues are different. It is not part of this  PR though, as it should be treated separately.

**Comment by thekaveman** [comment]
> I'm starting this new thing of having a basic naming convention for images: (name of page)-(type of image)-(slug), like home-logo-calsta or partners-background-cdt.

:100: :100: :100:

**Comment by ohmegasquared** [comment]
should we add mobile/web in the image name convention?

**Comment by thekaveman** [comment]
@machikoyasuda could you please `rebase` this PR on `main` to get the other homepage changes in here, so we can see the final final designs with Partners?

**Comment by machikoyasuda** [comment]
@thekaveman Rebased and updated partner logos to use the latest (have been updated to have all the same height).

File: src/partners.html
Comment (thekaveman) [inline src/partners.html:34]: Vertical spacing between elements should be 24px Desktop / 16px Mobile according to Figma.

The preceding `<p>` tag has a 16px bottom margin applied, so when the top margin is added to this `<figure>` the spacing goes beyond 24px.

Recommend:

* Remove `<figure>` top-margin for Mobile
* Only add 8px `<figure>` top margin for Desktop

File: src/partners.html
Comment (thekaveman) [inline src/partners.html:68]: Same issue here with too much vertical spacing, since the `<ul>` has a bottom margin of 16px already.

**Review by thekaveman — CHANGES_REQUESTED** [review]
Some small spacing issues

File: src/partners.html
Comment (machikoyasuda) [reply src/partners.html:34]: The preceding `<p>` tag's 16px bottom margin actually doesn't add to the `<figure>`s top margin. The yellow area below is 16px: 


image

Unless maybe this is a Linux difference (???)

Comment (machikoyasuda) [reply src/partners.html:34]: Just checked on Chrome + Linux and it has the same behavior as Firefox/Chrome + Mac does for me -- the margins aren't cumulative.

Comment (thekaveman) [reply src/partners.html:34]: Yeah, you are right -- they both have margin in that space, but they aren't being added together. Thanks!

Comment (machikoyasuda) [reply src/partners.html:34]: Yeah it's confusing!!!!

**Comment by machikoyasuda** [comment]
🚧 🚧 🚧 
FYI - This pull request is _not_ going to Production. We are still waiting on final copy and approvals.

The "reviews" at this stage for this PR are purely for the design from @ohmegasquared and code from @thekaveman. 

🚧 🚧 🚧

**Comment by machikoyasuda** [comment]
Planning to push up a cleaned up version of this PR after the smaller PRs are merged into `main`.

**Pull request closed without merging**
